tests/performance/live.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+set -e
+
+MARATHON_DIR=$(pwd)
+MARATHON_PERF_TESTING_DIR=$(pwd)/marathon-perf-testing
+
+# Bring an up-to-date marathon-perf-testing environment
+[ ! -d marathon-perf-testing ] && git clone "https://${GIT_USER}:${GIT_PASS}@github.com/mesosphere/marathon-perf-testing.git"
+(cd marathon-perf-testing; git pull --rebase)
+
+# Privileged clean-up of the results folder
+docker run -i --rm \
+    --privileged \
+    -v /var/run/docker.sock:/var/run/docker.sock \
+    -v "$MARATHON_DIR:/marathon" \
+    icharalampidis/marathon-perf-testing:latest \
+    rm -rf /marathon/results
+
+# Configuration
+DOCKER_NETWORK=testing
+
+# Get the git hash
+GIT_HASH=$(cd "$MARATHON_DIR" && git rev-parse HEAD)
+echo "- Running on GIT hash $GIT_HASH..."
+
+# Run the CI job
+docker run -i --rm \
+    --network ${DOCKER_NETWORK} \
+    --privileged \
+    -v /var/run/docker.sock:/var/run/docker.sock \
+    -v "$MARATHON_DIR:/marathon" \
+    -e "PARTIAL_TESTS=test-5k-apps" \
+    -e "PERF_DRIVER_ENVIRONMENT=env-ci-live.yml" \
+    -e "DATADOG_API_KEY=$DATADOG_API_KEY" \
+    -e "DATADOG_APP_KEY=$DATADOG_APP_KEY" \
+    -e "AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID" \
```

Comment:
This particular test does not need to upload ton S3, so you can safely ignore these environment variables. However they must be non-empty for the bootstrap validation to pass. Do something like this in the beginning:

```
AWS_ACCESS_KEY_ID=unused
AWS_SECRET_ACCESS_KEY=unused
```